meilisearch/src/extractors/payload.rs

```diff
@@ -71,3 +71,40 @@ impl Stream for Payload {
         }
     }
 }
+
+#[cfg(test)]
+mod tests {
+    use actix_http::encoding::Decoder as Decompress;
+    use actix_http::BoxedPayloadStream;
+    use bytes::Bytes;
+    use futures_util::StreamExt;
+    use meili_snap::snapshot;
+
+    use super::*;
+
+    #[actix_rt::test]
```

Comment:
I didn't see any other unit tests defined this way. I hope it is not a problem.

Reply:
It doesn't look like an issue ☁️